### PR TITLE
Minutes for 2019-09-13

### PR DESCRIPTION
**Pull request by Adimote**
Thanks for attending!

### CODE REVIEW

File: SR2020/2019-09-13.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+- Jenny Fletcher
+- Peter Law
+
+
+## Outcome of meeting with kit team & Trustees:
+- Trustees want us to fix a kickstart date
+- Andy Busse joined a meeting with the kit team, Tom Wheal (from kit team) will give us an estimate on the number of teams we can support in terms of number of kits next week.
+- We’ll be able to get the kits ready early October.
+- Should we confirm the kits by the end of the month?
+  - We shouldn’t confirm more than 30 teams (Peter).
```
Comment (PeterJCLaw) [inline SR2020/2019-09-13.md]: Please could we clarify that this is a guess based on the numbers of working kits we had issued last year and the current state of the kit checking.

I didn't mention at the time, but for clarity - this is also accounting for the fact that we didn't get back at all some (3 I think) of the kits we handed out.

File: SR2020/2019-09-13.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+- Kits (see above)
+- Venue (Not blocking)
+- Game (Not blocking)
+- Microgames
+
+We can announce without a game or microgames, so we can announce the 26th of October.
+
+- We should lock down the date after we have Cambridge confirmed.
+- *Action Point*: Jake will send the Kickstart announcement email.
+- *Action Point*: Jake will also modify the announcement for Kickstart.
```
Comment (PeterJCLaw) [inline SR2020/2019-09-13.md]: If these actions have been added as issues on this repo, please could you cross link them?

(note that if you group them at the bottom per the template, there's a script you can run which does the import for you and also edits the file to add the links)

File: SR2020/2019-09-13.md [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+
+## Outcome of meeting with kit team & Trustees:
+- Trustees want us to fix a kickstart date
+- Andy Busse joined a meeting with the kit team, Tom Wheal (from kit team) will give us an estimate on the number of teams we can support in terms of number of kits next week.
+- We’ll be able to get the kits ready early October.
+- Should we confirm the kits by the end of the month?
+  - We shouldn’t confirm more than 30 teams (Peter).
+  - We've had 28 apply so far
+
+## What’s blocking kickstart?
```
Comment (PeterJCLaw) [inline SR2020/2019-09-13.md]: nit: maybe clarify that this is about blockers to _announcing_ kickstart?

File: SR2020/2019-09-13.md [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+  - They’ll be done by Tomorrow’s meeting (Thalistair)
+  - They’ll be discussed on Saturday (Tomorrow)
+  - We’ll present the principles and then decide which of the two best meet the criteria.
+- Hosting a ‘Doing’ weekend soon?
+  - Date: 28th September
+  - Throw date into volunteer email.
+  - Things to do:
+    - Run through microgames
+    - Identify a list of newbie tasks
+    - Make a game presentation
```
Comment (PeterJCLaw) [inline SR2020/2019-09-13.md]: Not sure this was suggested?

(I can maybe see advantage in doing run-throughs at a Doing, but I'm not sure that a Doing is conducive to actually creating the presentation)

**Comment by Adimote** [comment]
merge please :)

**Pull request merged**